src/components/OntologyBrowser/store.ts

```diff
@@ -106,14 +117,18 @@ export class PhenotypeStore {
     return [...this.tree];
   };
 
-  buildPhenotypeQuery = (field: string, filterThemselves: boolean) => `query($sqon: JSON) {
+  buildPhenotypeQuery = (
+    field: string,
+    filterThemselves: boolean,
+  ) => `query($sqon: JSON, $term_filters: JSON) {
     participant {
       aggregations(filters: $sqon, aggregations_filter_themselves: ${filterThemselves}) {
-        ${field}__name {
+        ${field.replace('.', '__')}__name {
```

Comment:
would it make it easier to read if field.replace('.', '__') would be extracted into a function with a meaningful name? const transformStuff = (field) => field.replace('.', '__')